sql3/sql_complex_test.go

```diff
@@ -2631,3 +2638,216 @@ func TestPlanner_BulkInsert_FB1831(t *testing.T) {
 		t.Fatal("Expecting to be equal")
 	}
 }
+
+type tb struct {
+	Name  string
+	Type  arrow.DataType
+	Value interface{}
+}
+
+func simpleParquetMaker(f *os.File, numRows int64, input []tb) error {
```

Comment:
we could make this a `t.Helper()`, use `assert.NoError(t, err)` within, and don't bother returning an error on this helper function